tiledb/multirange_indexing.py

```diff
@@ -128,14 +128,23 @@ def __getitem__(self, idx):
         attr_names = tuple(self.schema.attr(i)._internal_name for i in range(self.schema.nattr))
         coords = None
 
-        # default TILEDB_UNORDERED for sparse, TILEDB_ROW_MAJOR for dense
-        order = 'U' if self.schema.sparse else 'C'
+        # set default order
+        # - TILEDB_UNORDERED for sparse
+        # - TILEDB_ROW_MAJOR for dense
+        if self.schema.sparse:
+            order = 'U'
+        else:
+            order = 'C'
```

Comment:
Isn't this column major? The comment says row?

Reply:
This is numpy naming semantics, `C` is "C ordering" == "row major", `F` is "fortran ordering" == "row major".